examples/shared/node/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:21.5-bookworm-slim@sha256:9df21707e27d98ee83e079fa47c5c0c5fba6c768e566643f8944b00b00afc4a2 as node-base
+FROM node:21.6-bookworm-slim@sha256:e8a7eb273ef8a9ebc03f0ad03c0fd4bbc3562ec244691e6fc37344ee2c4357d2 as node-base
```

Comment:
why are these changes inhere?

Reply:
I messed up and rebased instead of merging. I believe I've fixed this (by merging). The commits are still there but the diff (in "files changed") looks correct now.